templates/aws/single_node/terraform/pools/main.tf

```diff
@@ -0,0 +1,52 @@
+terraform {
+  required_version = ">= 0.13.1"
+  required_providers {
```

Comment:
Is there a reason you're not using this template for a single node? https://github.com/rancherlabs/corral-packages/blob/main/templates/aws/nodes/manifest.yaml 
This template can be used to create a single. Let's prevent creating terraform packages that we don't need.

Reply:
updated